core/modules/menu/menu.admin.inc

```diff
@@ -403,7 +403,7 @@ function menu_edit_item($form, &$form_state, $type, $item, $menu) {
     '#type' => 'checkbox',
     '#title' => t('Show as expanded'),
     '#default_value' => $item['expanded'],
-    '#description' => t('If selected and this menu link has children, the menu will always appear expanded.'),
+    '#description' => t('When displayed in a 'hierarchial tree' menu style, children of expanded menu links will always be visible.'),
```

Comment:
The single quotes are causing a problem. I believe that you will need to use double quotes for "hierarchial tree" or for the other statement. 

Current PR is generating this error:
![image](https://user-images.githubusercontent.com/3144571/159835366-3795925c-a33e-4145-8708-9c5fe18229bb.png)

Reply:
@quicksketch or @BWPanda - I got this syntax error by simply trying to access 'admin/structure/menu'.

Does it indicate a lapse in test coverage that this page is not working and the tests did not catch it?

Reply:
@quicksketch and @BWPanda - I take it back, I saw green and thought the tests had run. But, now I look closer and it appears that tests did not run at all. That is a different problem.

Reply:
Tests probably didn't run because of the syntax error...

Reply:
```suggestion
    '#description' => t("When displayed in a 'hierarchial tree' menu style, children of expanded menu links will always be visible."),
```